tests/CustomElements/v1/js/customElements.js

```diff
@@ -313,6 +313,34 @@ suite('customElements', function() {
     done();
   });
 
+  test('customElements.get', function (done) {
+    class XBoo extensd HTMLElement {}
+    customElements.define('x-boo-get', XGetTest);
+    assert.equal(XGetTest, customElements.get('x-boo-get'));
+    done();
+  });
+
+  test('attributeChangedCallback for existing observed attributes', function (done) {
+    var changed = [];
+    class XBoo extends HTMLElement {
+      static observedAttributes () {
+        return ['test1'];
+      }
+      attributeChangedCallback(name, oldValue, newValue) {
+        changed.push({ name, oldValue, newValue });
+      }
+    }
+    var xboo = new XBoo();
+    xboo.setAttribute('test1', 'test1');
+    xboo.setAttribute('test2', 'test2');
+    customElements.flush();
```

Comment:
This seemed to be how other tests were initiating an upgrade pass. I'll get the tests running and then correct this if need be.